cli/tools/test/reporters/dot.rs

```diff
@@ -0,0 +1,207 @@
+// Copyright 2018-2023 the Deno authors. All rights reserved. MIT license.
+
+use super::common;
+use super::fmt::to_relative_path_or_remote_url;
+use super::*;
+
+pub struct DotTestReporter {
```

Comment:
What happens when someone does something like `console.log` in a test? Does that output get captured and displayed?

Reply:
> User output is silenced and not printed to the console.